types/html-to-text/html-to-text-tests.ts

```diff
@@ -209,3 +210,33 @@ console.log(htmlToText("<h1>Starting foo test</h1><foo>bar</foo>", {
         },
     ]
 }));
+
+console.log('Test with linkBrackets false');
+console.log(htmlToText("<a href=\"https://github.com/DefinitelyTyped\">Link</a>"), {
```

Comment:
I ran the tests and they all need to be corrected.  The closing `)` on this line should be on line 222.  The selectors are not being passed to `html-to-text`.

Reply:
@ekremkenter Any chance you can update the tests so we can close this out?